core/src/main/java/com/linecorp/armeria/server/docs/DocServiceBuilder.java

```diff
@@ -248,6 +258,82 @@ public DocServiceBuilder exampleRequest(Iterable<?> exampleRequests) {
         return this;
     }
 
+    /**
+     * Sets the {@link Predicate} that checks whether a service will be <b>included</b> while building
+     * {@link DocService}. The {@link Predicate} will be invoked with the service name.
+     * If this method and {@link #includeMethod(BiPredicate)} is not invoked, {@link DocService} will include
+     * all the services and methods it finds while generating documentation.
+     *
+     * <P>Note that this can be called multiple times and the {@link Predicate}s are composed using
+     * {@link Predicate#or(Predicate)}.
+     *
+     * @see #includeMethod(BiPredicate)
+     * @see DocService to find out how {@link DocService} generates documentaion
```

Comment:
Could you try to generate Javadoc to make sure this in rendered as expected?

Reply:
Oops, my bad. 😭